lib/ModelMixins/TableMixin.ts

```diff
@@ -376,14 +376,16 @@ function TableMixin<T extends AbstractConstructor<BaseType>>(Base: T) {
           : xColumn.valuesAsNumbers.values;
 
       const xAxis: ChartAxis = {
+        name: xColumn.title || xColumn.name,
         scale: xColumn.type === TableColumnType.time ? "time" : "linear",
         units: xColumn.units
       };
 
       return filterOutUndefined(
         lines.map((line) => {
+          const yColumnId = line.yAxisColumn;
           const yColumn = this.findColumnByName(line.yAxisColumn);
-          if (yColumn === undefined) {
+          if (yColumnId === undefined || yColumn === undefined) {
             return undefined;
           }
```

Comment:
Same as before but just convinces TS that value is not `undefined` when using `yColumnId` below.